client/src/Movies/MovieList.js

```diff
@@ -32,23 +34,8 @@ export default class MovieList extends Component {
 }
 
 function MovieDetails({ movie }) {
-  const { title, director, metascore, stars } = movie;
   return (
-    <div className="movie-card">
-      <h2>{title}</h2>
-      <div className="movie-director">
-        Director: <em>{director}</em>
-      </div>
-      <div className="movie-metascore">
-        Metascore: <strong>{metascore}</strong>
-      </div>
-      <h3>Actors</h3>
-
-      {stars.map(star => (
-        <div key={star} className="movie-star">
-          {star}
-        </div>
-      ))}
-    </div>
-  );
+  <div className = 'movie-list'>
+    <NavLink to = {`/movies/${movie.id}`} style={{textDecoration: 'none', color: "black"}} > <MovieCard movie = {movie} />  </NavLink> 
```

Comment:
you could also style this easily with css or styled components.. when ever you don't know what an element is, just inspect it in your chrome dev tools. Try not to default to setting the styles in line as that can be really hard to debug or change later.